eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -16,7 +16,7 @@
 
 package fr.acinq.eclair
 
-import fr.acinq.bitcoin.scalacompat.{Block, ByteVector32, Satoshi, SatoshiLong, Script}
+import fr.acinq.bitcoin.scalacompat.{Block, BtcDouble, ByteVector32, Satoshi, SatoshiLong, Script}
```

Comment:
What is `BtcDouble` used for?

Reply:
Good catch, it's not necessary right now, I should remove it!
It contains syntax sugar to write `2 btc`, but I'm not using it here.
This PR isn't ready to review though, I'd like to wait for dual-funding to be integrated before merging it, as there may be a subtlety when both peers contribute to the channel.